build.gradle

```diff
@@ -697,3 +708,16 @@ task installDist(overwrite: true) {
     // This task must be overwritten to do as it's breaking JAR packaging and it's a mandatory step of swagger
     // documentation publication.
 }
+
+task executeFeatures(type: JavaExec, dependsOn: testClasses) {
```

Comment:
This could be achieved by a runner class annotated with these values as parameters. I think that could be better for maintenance purpose.

Reply:
Move this bit of code into functional task.